client/post-editor/editor-featured-image/preview-container.jsx

```diff
@@ -84,9 +83,9 @@ export default class extends React.Component {
 
 			if ( image && image.ID !== this.props.itemId ) {
 				// TODO: REDUX - remove flux actions when whole post-editor is reduxified
-				PostActions.edit( {
-					featured_image: image.ID,
-				} );
+				// PostActions.edit( {
+				// 	featured_image: image.ID,
+				// } );
```

Comment:
Remove rather than comment out?

Reply:
Thanks @nylen for the reminder about this "todo". I converted this Flux dispatch to a Redux one in https://github.com/Automattic/wp-calypso/pull/25160/commits/b7d8e28126f138276c2d3fa1c90493219e529739.

The dispatch is necessary for several cases to work correctly:
- uploading a new image to media library and choosing it as featured img before the upload is complete. This ensures that the transient ID is updated to the permanent one.
- setting featured image by selecting it from Google library. Again, transient ID vs permanent

The Google library case was actually reported as an issue recently in #24807. I forgot to update this one from Flux to Redux when migrating the Featured Image component. I verified that the patch fixes the issue.